src/main/resources/add-ons-to-index.json

```diff
@@ -131,7 +137,7 @@
       "type": "OMOD",
       "name": "Address Hierarchy Rwanda",
       "tags": [
-        "pih-rwanda"
+        "pih-rwanda", "Address"
```

Comment:
Taking part of the module name and including it as a tag isn't really helpful, kindly try to add meaningful tags.